src/main.ts

```diff
@@ -46,7 +46,7 @@ export async function run() {
     }
 
     if (cache) {
-      if (isGhes()) {
+      if (!isCacheFeatureAvailable()) {
         throw new Error('Caching is not supported on GHES');
```

Comment:
Let's not throw the error here (if we can throw it from isCacheFeatureAvailable function ). We can return if the Cache feature is not available

Reply:
Done. Thank you.